Makefile

```
@@ -1,5 +1,11 @@
-.PHONY: generate-protos
+.PHONY: generate-protos setup
```

```suggestion
.PHONY: generate-protos setup format
```

The `format` command must be added as phony to be accessible from the terminal.